Test/testNS1.cpp

```diff
@@ -0,0 +1,45 @@
+/* testNS1.cpp continuously reads photodiode data and transmits each data point via serial
+ * Usage:
+ *  This is a quick and dirty script based on the original NS1 software to test the basic functionality of NanoSam I
+ *  Run this script through Teensyduino to test the NS1 payload, monitor via the serial monitor/plot
+ *  
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+#include <SPI.h>
+
+/* - - - - - - Initialization - - - - - - */
+const int ADC_CHIP_SELECT = 10; //chip select pin number for ADC
+
+/* - - - - - - Functions - - - - - - */
+
+// getADC reads and returns one 16 bit data point from the ADC via SPI
+uint16_t getADC()
```

Comment:
do we always have to use this custom defined uint16_t instead of float when dealing with the ADC? more to know for myself in the future

Reply:
I'm not sure, both are 16 bits so I would assume both are acceptable, if for some reason we have to use uint16 to read the SPI we can convert it to a float to work with the code you already wrote.